ERCS/erc-xx.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
```suggestion
---
eip: 8064
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.